### ISSUE TITLE
[feature] Make the jinja2 Templates import/include load path consistent with the normal profile include load path

### ISSUE THREAD

**Issue by todorico** [conan-io/conan#17431]
### What is your suggestion?

To illustrate the issue, let's consider the following profile hierarchy:

```
profiles/
    subdirA/profileA
    subdirB/profileB
```

If I want to include `profileA` in `profileB`, it is currently impossible to do so using Jinja syntax because the import paths seem to be relative to the current profile. For example:

**In profileB:**

```jinja
{% include "subdirA/profileA" %} # Not Working
{% import "subdirA/profileA" as vars %} # Not Working
```

However, if I use the following syntax, it works because the import path is relative to the root profiles directory:

```jinja
include(subdirA/profileA) # Working
```

Currently, it is not possible to import the profile from a neighboring directory using Jinja syntax. Would it be possible to allow Jinja templates to be loaded from the root profiles directory? This change would provide greater flexibility in organizing my profiles.

Perhaps the following function could be useful: [Jinja2 FileSystemLoader](https://jinja.palletsprojects.com/en/stable/api/#jinja2.FileSystemLoader).

Thank you for considering this enhancement!

### Have you read the CONTRIBUTING guide?

- [x] I've read the CONTRIBUTING guide

**Comment by memsharded** [comment]
Hi @todorico 

Thanks for your suggestion.

I am trying to implement it in https://github.com/conan-io/conan/pull/17432, I think it might be doable, but we need to double check it won't be breaking other users.

**Comment by todorico** [comment]
Awesome, I really appreciate your effort! 😄

**Comment by memsharded** [comment]
https://github.com/conan-io/conan/pull/17432 merged, this will be in next Conan 2.11, thanks for the feedback!
